### PR TITLE
Update dynamic-theme-fixes.config (myacademy.oracle.com)

### PR DESCRIPTION
**Pull request by AlyaqdhanZ**
Fixed dark theme for oracle academy and the slides player

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -16363,6 +16363,33 @@ body {
...
+.logo--desktop
+.login-logo
+#logo
+
+CSS
+header, #header {
+    background: #333 !important;
+}
+.message-box__icon.message-box__icon_type_question, .splide__arrow {
+    background-color: white !important;
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:16383]: Similar comment as above:
`background-color: ${black} !important;`

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -16363,6 +16363,33 @@ body {
...
+    border-radius: 50%;
+    border: 2px solid white !important;
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:16385]: `border: 2px solid ${black} !important;`

**Review by Myshor — CHANGES_REQUESTED** [review]
Some suggestions in review comments.

One more request.
Can you share the screens to show how it was before fix and after in dark scheme?
+ check how it will look after changing to light scheme.

myacademy.oracle.com needs account to browse there and without I can only approve looking on screens.

**Comment by AlyaqdhanZ** [comment]
before fix:
image
image

dark theme (fixed):
image
image

light theme (fixed):
image
image

also I think people won't use the extension light theme they would just disable it, at least that's what I do.

**Comment by Myshor** [comment]
Some use light theme and some contrast + brightness changes... I know they use because I had some bugs about that already. Not many but uses. 🙂

**Review by Myshor — APPROVED** [review]

**Comment by Myshor** [comment]
Fixes merged.

**Pull request merged**